openslides/mediafiles/static/js/mediafiles/list.js

```diff
@@ -22,8 +22,9 @@ angular.module('OpenSlidesApp.mediafiles.list', [
     'User',
     'Mediafile',
     'MediafileForm',
+    'Logos',
```

Comment:
You should add `'OpenSlides.mediafiles.logo'` also to the module import list because we directly import `Logos` here. This is – of course – for readability only.